lib/licenses.nix

```diff
@@ -545,6 +545,13 @@ lib.mapAttrs mkLicense ({
     free = false;
   };
 
+  firefox = {
+    fullName = "Firefox Terms of Use";
+    url = "https://www.mozilla.org/about/legal/terms/firefox/";
+    free = false;
```

Comment:
It's unfortunate, but I agree that the clause you mentioned in a Freedom 0 violation.